std/bigint.d

```diff
@@ -277,8 +277,8 @@ public:
 
     // BigInt op integer
     BigInt opBinary(string op, T)(T y) pure nothrow const
-        if ((op=="+" || op == "*" || op=="-" || op=="/" || op=="|" || op=="&" ||
-            op=="^"|| op==">>" || op=="<<" || op=="^^")
+        if ((op=="+" || op == "*" || op=="-" || op=="/" || op=="%" || op=="|" ||
+            op=="&" || op=="^" || op==">>" || op=="<<" || op=="^^")
```

Comment:
The interesting thing about modulo is that a BigInt modulo a long/ulong can return a long/ulong (the result must fit by definition of modulo). Can you separate out the implementation to do this?